src/Core/Pds/PdsService.cs

```diff
@@ -241,27 +240,30 @@ private void EnrichFhirBundleWithPatientsToBeDeleted(List<string> patientsToBeDe
         }
     }
 
-    private string HandleInvalidPDSPatients(Result<string> csvToJsonResult, List<string> patientsToBeDeleted)
+    private string HandleInvalidPDSPatients(string csvToJsonResult, List<string> patientsToBeDeleted)
```

Comment:
Sorry for doing this here but can you please rename the PDS to Pds so it will be consistent through the code base?

Reply:
It is disturbing me in so many different levels